hyperion-core/src/main/java/com/willowtreeapps/hyperion/core/Hyperion.java

```diff
@@ -39,20 +39,44 @@ public static void setShakeGestureSensitivity(float sensitivity) {
 
     /**
      * Manually trigger the Hyperion menu embedded in the current foreground {@link Activity}
+     *
+     * @return true if the menu was opened, otherwise false
      **/
-    public static void open() {
+    public static boolean open() {
         requireApplication();
-        AppComponent.Holder.getInstance(application).getPublicControl().open();
+        return AppComponent.Holder.getInstance(application).getPublicControl().open();
     }
 
     /**
      * Manually trigger the Hyperion menu embedded in the given {@link Activity} to open.
      *
      * @param activity the {@link Activity} containing the menu to open.
+     * @return true if the menu was opened, otherwise false
+     */
+    public static boolean open(Activity activity) {
+        requireApplication();
+        return AppComponent.Holder.getInstance(application).getPublicControl().open(activity);
+    }
+
+    /**
+     * Manually trigger closing the Hyperion menu embedded in the current foreground {@link Activity}
+     *
+     * @return true if the menu was closed, otherwise false
+     **/
+    public static boolean close() {
+        requireApplication();
+        return AppComponent.Holder.getInstance(application).getPublicControl().close();
+    }
+
+    /**
+     * Manually trigger the Hyperion menu embedded in the given {@link Activity} to close.
+     *
+     * @param activity the {@link Activity} containing the menu to close.
+     * @return true if the menu was opened, otherwise false
```

Comment:
I think this should be `true if the menu was closed, otherwise false`

Reply:
Blah, yep, copypaste error